text/0134-stats-api.md

```diff
@@ -0,0 +1,172 @@
+# Stats API
+
+## 1. Summary
+
+This specification describes the stats API endpoints.
+
+Stats routes give information and metrics about indexes and the Meilisearch database.
+
+## 2. Motivation
+N/A
+
+## 3. Functional Specification
+
+### 3.1. API Endpoints Definition
+
+See statistics of Meilisearch indexes.
+
+- [3.1.1. `GET` - `stats`](#311-get---stats)
+- [3.1.2. `GET` - `indexes/:index_uid/stats`](#312-get---indexesindexuidstats)
+
+#### 3.1.1. `GET` - `stats`
+
+Get stats for all indexes.
+
+##### 3.1.1.1. Path Parameters
+
+| Parameter               | Type                     | Required |
+|-------------------------|--------------------------|----------|
+| `index_uid`             | String                   | true     |
+
+
+###### 3.1.1.1.1. `index_uid`
+
+- Type: String
+- Required: True
+
+Unique identifier of an index.
+
+##### 3.1.1.2. Response Definition
+
+In addition to all fields returned by [3.1.2. `GET` - `indexes/:index_uid/stats`](#312-get---indexesindexuidstats), this route returns the instance-level fields.
```

Comment:
I think it would be better if 3.1.1. is  `GET` - `indexes/:index_uid/stats` and 3.1.2 is`GET` - `stats`. That way the reader won't have to scroll to 3.1.2 to understand 3.1.1